cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -945,6 +950,39 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
                 },
             };
         }
+        //Added Remove Annotations in Range
+        case AnnotationActionTypes.REMOVE_ANNOTATIONS_INRANGE_SUCCESS: {
+            const { sessionInstance, startFrame, endFrame } = action.payload;
+            return {
+                ...state,
+                removeinrange: {
+                    ...state.removeinrange,
+                    startFrame,
+                    endFrame,
+                },
+            };
+        }
+        case AnnotationActionTypes.REMOVE_ANNOTATIONS_INRANGE: {
+            const { sessionInstance } = action.payload;
+            return {
+                ...state,
+                removeinrange: {
+                    ...state.removeinrange,
+                    sessionInstance,
```

Comment:
Why do we need this field in redux storage? It is already there. But in your case it is not necessary even to pass to ``removeAnnotationsinRangeAsync``. Just look how ``fetchAnnotationsAsync`` is implemented.